DEV-4/src/abstractClasses/Counter.java

```diff
@@ -0,0 +1,25 @@
+package abstractClasses;
+
+/**
+ * Abstract class which provides his methods to _Counter classes
```

Comment:
_Counter classes? What does it mean? if you meant successors than you must write successors not _Counter classes